src/sessions.ts

```diff
@@ -58,8 +59,11 @@ export interface ClientSessionOptions {
   snapshot?: boolean;
   /** The default TransactionOptions to use for transactions started on this session. */
   defaultTransactionOptions?: TransactionOptions;
-  /** @internal
-   * The value of timeoutMS used for CSOT. Used to override client timeoutMS */
+  /**
+   * @public
+   * An overriding timeoutMS value to use for a client-side timeout.
```

Comment:
Should this be marked experimental, too? (If so, we should mention it in our revisit experimental tags ticket)

Reply:
addressed in https://github.com/mongodb/node-mongodb-native/pull/4315/files